bitcoin/src/blockdata/transaction.rs

```diff
@@ -200,7 +200,7 @@ pub struct TxIn {
 
 impl TxIn {
     /// The weight of a `TxIn` excluding the `script_sig` and `witness`.
-    pub const BASE_WEIGHT: Weight = Weight::from_wu(32 + 4 + 4);
+    pub const BASE_WEIGHT: Weight = Weight::from_vb_const(32 + 4 + 4);
```

Comment:
Can you add a code comment for the magic numbers please, since we got this wrong once already.